openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -205,33 +205,50 @@ def create_instance(creator_name, instance_name, **options):
     )
 
 
-def load_casting(project_name, shot_name) -> Set[OpenpypeContainer]:
-    """Load casting from shot_name using kitsu api."""
+def download_kitsu_casting(
+    project_name: str, shot_name: str, rig_only: bool = False
```

Comment:
I'm sorry, but your code gives me ideas :laughing: 

Instead of `rig_only`, maybe introduce a filtering system, like `asset_types: List[str]`, which you'll match in the loop below. If `None` specified, use all, else only the given ones. `"Character"` only in your case.

Reply:
No need to apologize, that's the whole point of working collectively ^^
I think it's a good idea, that would make the `download_kitsu_casting` method more reusable.